src/utils.ts

```diff
@@ -1,17 +1,19 @@
 export const getRotationDegrees = (
   prizeNumber: number,
-  numberOfPrizes: number
+  numberOfPrizes: number,
+  randomDif = true
 ) => {
   const degreesPerPrize = 360 / numberOfPrizes;
 
   const initialRotation = 43 + degreesPerPrize / 2;
 
   const randomDifference = (-1 + Math.random() * 2) * degreesPerPrize * 0.35;
 
-  const prizeRotation =
-    degreesPerPrize * (numberOfPrizes - prizeNumber) -
-    initialRotation +
-    randomDifference;
+  const prizeRotation = randomDif
+    ? degreesPerPrize * (numberOfPrizes - prizeNumber) -
+      initialRotation +
+      randomDifference
+    : degreesPerPrize * (numberOfPrizes - prizeNumber) - initialRotation;
```

Comment:
This is a bit confusing. I would create two variables before the assignment.

One variable should be the result of: `degreesPerPrize * (numberOfPrizes - prizeNumber) - initialRotation + randomDifference`

And the other should be the result of: `degreesPerPrize * (numberOfPrizes - prizeNumber) - initialRotation`